api/applicationserver.proto

```diff
@@ -106,6 +106,15 @@ service As {
   };
 }
 
+message NsAsHandleUplinkRequest {
```

Comment:
I think that `ApplicationUps` is more consistent (i.e. we have `Clients` which has `repeated Client`)

Reply:
The reasoning behind the naming was that later we may want to add other fields to this request, so `*Request` fits this better in that case.

Reply:
We can do that later and stay wire compatible. But I'm fine with either